build.gradle

```diff
@@ -137,6 +137,22 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Huy Le"
+                        email = "huy.lenq@gmail.com"
+                    }
```

Comment:
@huylenq Let us know when you don't want to be nominated there. (I assumed you're okay as you were also okay with `embulk-input-s3`.)